image_topic: '/spinnaker_ros_driver_node/cam_fm_01/image_raw'
image_output_path: '/home/baykara/MST/IMG'
canbus_topic: '/canbus'
csv_output_path: '/home/baykara/MST/'
csv_file_name: 'data1.csv'
